docs/reference/naming-suggestions.rst

```diff
@@ -0,0 +1,10 @@
+Naming Suggestions for Open edX Events
+########################################
+
+When naming a new event and contributing it back to this repository, consider the following suggestions:
+
+- Use a name that is descriptive to the event's purpose. For example, the event associated with a user's enrollment in a course is named ``COURSE_ENROLLMENT_CREATED``, which clearly indicates the event's purpose.
```

Comment:
Do you think the name should always have a verb in it? Generally would it always be a verb in past tense eg created, enrolled, succeeded, failed?

Reply:
In this commit I specified where this standard we've enforced so far comes from: https://github.com/openedx/openedx-events/pull/465/commits/34562b174b8c90745940538b2152ea57570753a1. But now that you mention it, I think it'd be a good idea to emphasize it here as well. Thanks!